src/img/icons--check.svg

```diff
@@ -1,3 +1,3 @@
-<svg id="check" viewBox="0 0 22 16" xmlns="http://www.w3.org/2000/svg">
+<svg id="check" viewBox="-1 0 22 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We shouldn't have to adjust this, but as this is the only place this icon is used, I would match it up to the exact height used
```suggestion
<svg id="check" viewBox="0 0 14 10" xmlns="http://www.w3.org/2000/svg">
```

Reply:
Changing this makes the tick far too big and then doesn't fit in the box. If I make all the other changes but leave this one then it looks correct. So ill make the other changes and you can re-review to see what you think